shell/plugins/steve/actions.js

```diff
@@ -18,6 +18,8 @@ export default {
   async request({ state, dispatch, rootGetters }, pOpt ) {
     const opt = pOpt.opt || pOpt;
 
+    opt.params = { exclude: 'metadata.managedFields' };
```

Comment:
We should add to rather than replace the opt.params object.

Looking at their [spec ](https://github.com/rancher/rancher/issues/36683#issuecomment-1102884686) the request can contain multiple `exclude`s, i'm not sure how we make this safe for requests that pass in their own `exclude` value.

The spec states it's also only applicable to the `steve` / `v1` api. One way to do this could be to add the param in the steve getter `urlOptions`. This is called by `urlFor` in the `find` actions to create the url passed to the request action. It might be easier to check the url for the v1 there, rather than having to do some smart parsing to ensure the v1 in kube api urls isn't a false positive. It would also be easier to add multiple excludes in that context (it sticks all params on the url.

So i guess
- Blocking - opt.params should not be overwritten
- Blocking - Param should only be applied to steve/v1 requests
- Nice to have - Handle exclude param/s passed through to request action